aws/sdk/build.gradle.kts

```diff
@@ -239,6 +241,17 @@ task("generateCargoWorkspace") {
     outputs.upToDateWhen { false }
 }
 
+tasks.register<Exec>("fixManifests") {
+    description = "Run the publisher tool's `fix-manifests` sub-command on the generated services"
+    workingDir(rootProject.projectDir.resolve("tools/publisher"))
+    commandLine("cargo", "run", "--", "fix-manifests", "--location", outputDir.absolutePath)
```

Comment:
I think this needs `outputDir` registered as an output and `tools/publisher` registered as an input